modules/contacts/contacts.service.js

```diff
@@ -0,0 +1,21 @@
+const Contact = require('./contacts.model');
+
+class ContactsService {
+  async getContacts() {
+    return await Contact.find();
+  }
+
+  async addContact(data) {
+    return new Contact(data).save();
+  }
+
+  async updateContact(id, contact) {
+    return await Contact.findByIdAndUpdate(id, contact);
+  }
+
+  async deleteContact(id) {
+    return await Contact.findByIdAndDelete(id);
+  }
+}
+
```

Comment:
you have to make checks in services and throw errors if there is something wrong. then you will catch it in resolver. For example, here you can check if it is wrong "id" and throw an error "CONTACT_NOT_FOUND". Check "news" or "products" services